tests/test_wave_01.py

```diff
@@ -119,12 +119,16 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    # raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    
+    # this is my own created assertion, is this right?
+
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
```

Comment:
This assertion looks great! Just to cover all our bases, we might want to add an assertion that checks the genre and rating as well!